meta-aos-tune/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend

```diff
@@ -0,0 +1,29 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Same for this recipe:
```sh
FILESEXTRAPATHS_prepend := "${THISDIR}/files:"

AOS_IAM_IDENT_MODULES = "\
    identhandler/modules/visidentifier \
"

DEPENDS_${PN} += " \
    aos-provfirewall \
"

pkg_postinst_${PN}() {
    # Add aosiam to /etc/hosts
    if ! grep -q 'aosiam' $D${sysconfdir}/hosts ; then
        echo '127.0.0.1	aosiam' >> $D${sysconfdir}/hosts
    fi
}
```

Reply:
no, the config file differs from the original one